pages/tx/[hash].tsx

```diff
@@ -492,6 +510,86 @@ const Tx = () => {
     },
   ]
 
+  const userOperations: InfoItermProps[] = [
+    {
+      field: t('call_contract'),
+      content: tx?.polyjuice?.call_contract ? (
+        <HashLink label={tx.polyjuice.call_contract} href={`/address/${tx.polyjuice.call_contract}`} />
+      ) : (
+        '-'
+      ),
+    },
+    { field: t('call_gas_limit'), content: tx?.polyjuice?.call_gas_limit || '-' },
+    { field: t('verification_gas_limit'), content: tx?.polyjuice?.verification_gas_limit || '-' },
+    {
+      field: t('paymaster'),
+      content: tx?.polyjuice?.paymaster_and_data ? (
+        <HashLink
+          label={tx.polyjuice.paymaster_and_data.slice(0, 20)}
```

Comment:
Please notice that `paymaster and data` is returned as a hex string, 2 chars in a hex string is a byte, so 42 chars are required(2 for `0x` and 40 for 20 bytes)